torch_brain/data/__init__.py

```
@@ -13,3 +13,46 @@
+    "collate",
+    "pad",
+    "pad8",
+    "pad2d",
+    "pad2d8",
+    "track_batch",
+    "track_mask",
+    "track_mask8",
+    "track_mask2d",
+    "track_mask2d8",
+    "Dataset",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)